### ISSUE TITLE
Error with Complex Particles

### ISSUE THREAD

**Issue by arbenede** [baggepinnen/MonteCarloMeasurements.jl#66]
Hello everybody, it is still me... According to the MonteCarloMeasurements.jl paper it seems that Complex Particles are supported, however I am getting an error, see this MWE:

```
using MonteCarloMeasurements
using Distributions

Y = [Particles(100, Normal(10+3i,2)) for i=1:3]

a = sin(2π*Y[1])
b = cos(2π*Y[2])

@show typeof(a)
@show typeof(a)

z = a + im*b
```

gives

```
typeof(a) = Particles{Float64,100}
typeof(a) = Particles{Float64,100}
Comparison operators are not well defined for uncertain values and are currently turned off. Call `unsafe_comparisons(true)` to enable comparison operators for particles using the current reduction function mean. Change this function using `set_comparison_function(f)`.

Stacktrace:
 [1] error(::String) at .\error.jl:33
```

Are Complex Particles still not fully supported maybe?

**Comment by baggepinnen** [comment]
Would you please write the full stack trace? I have a feeling the error occurs in the display of the number, not the use of it. It's still a bug that needs to be solved though.

**Comment by baggepinnen** [comment]
Note to myself: maybe a complex particle type would greatly simplify the support for complex numbers. `ComplexParticles < Complex`

**Comment by baggepinnen** [comment]
Fixed in 5ff43a4b3177a29c5ad773aae46406e5259f8fb9
New release tagged https://github.com/JuliaRegistries/General/pull/9919
